### PR TITLE
Issue #2: Reading conll u files doesn't work

### PR DESCRIPTION
**Pull request by VincTheSecond**
#2
I refactored the read block of the udapi-python to start works.
In addition, I fixed the bug in the write block as well so now the demo pipeline works.

### CODE REVIEW

File: udapi/core/bundle.py [hunk 1]
```diff
@@ -7,7 +7,12 @@
 from root import Root
 
 class Bundle(object):
-    """Bundle can be used for embracing two or more Universal Dependency trees that are associated in some way (e.g. parallel translations) inside a document. Unless different zones are differentiated in a bundle, there's only one tree per bundle by default."""
```
Comment (martinpopel) [inline udapi/core/bundle.py]: https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Comment (martinpopel) [inline udapi/core/bundle.py]: PEP008 says that for docstrings (and comments), the line length should be limited to 72 characters (and for other code 79 characters).
I would say that even 100 characters is ok (unless we want to strictly follow PEPs), but this line has 262 characters.

Comment (VincTheSecond) [inline udapi/core/bundle.py]: But this line comes from @zabokrtsky and my commit replaces this with several shorter lines :-)

Comment (martinpopel) [inline udapi/core/bundle.py]: Oh, sorry. I am fool.

**Pull request merged**